src/modules/swiper.js

```diff
@@ -0,0 +1,17 @@
+import Swiper,{ Navigation, Pagination} from 'swiper';
+import 'swiper/css';
+import 'swiper/css/pagination';
+const swiper = new Swiper('.swiper', {
```

Comment:
форматирование между импортами и кодом отступ

```suggestion
import 'swiper/css/pagination';

const swiper = new Swiper('.swiper', {
```